src/pages/networks/NetworkForwards.tsx

```diff
@@ -89,13 +93,27 @@ const NetworkForwards: FC<Props> = ({ network, project }) => {
         {
           content: (
             <>
-              <Link
-                className="p-button--base u-no-margin--bottom has-icon"
-                to={`/ui/project/${project}/network/${network.name}/forwards/${forward.listen_address}/edit`}
-                title="Edit network forward"
+              <Button
+                key="edit"
+                appearance="base"
+                className="u-no-margin--bottom"
+                dense
+                hasIcon
+                onClick={() =>
+                  void navigate(
+                    `/ui/project/${project}/network/${network.name}/forwards/${forward.listen_address}/edit`,
+                  )
+                }
+                type="button"
+                title={
+                  canEditNetwork(network)
+                    ? "Edit network forward"
+                    : "You do not have permission to edit forwards for this network"
+                }
+                disabled={!canEditNetwork(network)}
```

Comment:
This should stay a link so we can open it in a new tab and it behaves like a native link. Better switch it out for a disabled button in the read only case.